cmd/ui/src/utils.ts

```diff
@@ -40,21 +40,6 @@ export const getDatesInRange = (startDate: Date, endDate: Date) => {
     return dates;
 };
 
-export const validateNodeType = (type: string): ActiveDirectoryNodeKind | AzureNodeKind | undefined => {
```

Comment:
Thanks for consolidating this! We should remove the unused imports for `ActiveDirectoryNodeKind` and `AzureNodeKind` from this file now that validateNodeType has been moved